tools/rosetta/converter.go

```diff
@@ -115,7 +115,15 @@ func NewConverter(cdc *codec.ProtoCodec, ir codectypes.InterfaceRegistry, cfg sd
 		txDecode:        cfg.TxDecoder(),
 		txEncode:        cfg.TxEncoder(),
 		bytesToSign: func(tx authsigning.Tx, signerData authsigning.SignerData) (b []byte, err error) {
-			bytesToSign, err := cfg.SignModeHandler().GetSignBytes(context.TODO(), signing.SignMode_SIGN_MODE_LEGACY_AMINO_JSON, signerData, tx)
+			var bytesToSign []byte
+			h, ok := cfg.SignModeHandler().(authsigning.SignModeHandlerWithContext)
```

Comment:
If we copy the interface, then `ok` will never be true, because `cfg.SignModeHandler()` will be coming from the older version of the SDK.

In this PR, there are 2-way replace directives between main and rosetta. I think this will work, right? Probably simapp will need replace directives too in that start command PR.

Reply:
If we can get a few versions of rosetta out before forcing the SDK upgrade, I think it's fine indeed.

Reply:
I removed the replace directive, and just used the old method.

Reply:
👍🏾 I think we can remove it from simapp go.mod and SDK go.mod too then.